src/DI.Specification.Tests/DI.Specification.Tests.csproj

```diff
@@ -5,6 +5,8 @@
   <PropertyGroup>
     <Description>Suite of xUnit.net tests to check for container compatibility with Microsoft.Extensions.DependencyInjection.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.Extensions.DependencyInjection.Specification.Tests</AssemblyName>
+    <RootNamespace>Microsoft.Extensions.DependencyInjection.Specification</RootNamespace>
```

Comment:
Any reason to have this deviate? Normally if these two settings aren't set, their default value is the project's filename or something, no?